core/pta/system.c

```diff
@@ -226,7 +227,7 @@ static void ta_bin_close(void *ptr)
 	free(binh);
 }
 
-static TEE_Result system_open_ta_binary(struct system_ctx *ctx,
+static TEE_Result system_open_ts_binary(struct system_ctx *ctx,
```

Comment:
I think we can keep the name as `system_open_ta_binary()` for now.

Reply:
done